spec/frame_embed_generator_spec.rb

```diff
@@ -0,0 +1,12 @@
+require_relative '../_plugins/frame_embed_generator.rb'  
+require 'rspec'  
+require 'rack/test'
+
+set :environment, :test
+
+describe 'FrameEmbedGenerator Plugin Testing' do
+
+    include Rack::Test::Methods
+  
+    # TODO: Test
```

Comment:
Should probably change that :-)

Reply:
removed spec file, for reason explained in earlier comment.